markhor_flippers/src/markhor_flippers.cpp

```diff
@@ -32,40 +38,184 @@ void joyCallback(const sensor_msgs::Joy::ConstPtr& joy)
     flipper_rl_pub.publish(msg);
     flipper_rr_pub.publish(msg);
   }
+  if (individual_mode == true)
+  {
+    ROS_INFO("individual mode is successful"); 
+
+    if (flipper_mode_fl == true)
+    {
+      msg.data = joy->axes[4] * multiplicator;
+      flipper_fl_pub.publish(msg);
+    }
+    if (flipper_mode_fr == true)
+    {
+      msg.data = joy->axes[4] * multiplicator;
+      flipper_fr_pub.publish(msg);
+    }
+    if (flipper_mode_rl == true)
+    {
+      msg.data = joy->axes[4] * multiplicator;
+      flipper_rl_pub.publish(msg);
+    }
+    if (flipper_mode_rr == true)
+    {
+      msg.data = joy->axes[4] * multiplicator;
+      flipper_rr_pub.publish(msg);
+    }
+  }
 }
 
 bool flipperModeFrontEnable(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& res)
 {
   flipper_mode_front = true;
-  res.message = "successfully enable flipper mode front";
+  res.message = "successfully enabled flipper mode front";
   res.success = static_cast<unsigned char>(true);
   return true;
 }
 
 bool flipperModeFrontDisable(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& res)
 {
   flipper_mode_front = false;
-  res.message = "successfully disable flipper mode front";
+  res.message = "successfully disabled flipper mode front";
   res.success = static_cast<unsigned char>(true);
   return true;
 }
 
 bool flipperModeBackEnable(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& res)
 {
   flipper_mode_back = true;
-  res.message = "successfully enable flipper mode back";
+  res.message = "successfully enabled flipper mode back";
   res.success = static_cast<unsigned char>(true);
   return true;
 }
 
 bool flipperModeBackDisable(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& res)
 {
   flipper_mode_back = false;
-  res.message = "successfully disable flipper mode back";
+  res.message = "successfully disabled flipper mode back";
+  res.success = static_cast<unsigned char>(true);
+  return true;
+}
+
+bool diagnosticModeEnable(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& res)
+{
+  individual_mode = true;
+  flipper_mode_front = false;
+  flipper_mode_back = false;
+  res.message = "successfully enabled individual mode";
+  res.success = static_cast<unsigned char>(true);
+  return true;
+}
+
+bool diagnosticModeDisable(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& res)
+{
+  individual_mode = false;
+  flipper_mode_fl = false;
+  flipper_mode_fr = false;
+  flipper_mode_rl = false;
+  flipper_mode_rr = false;
+
+  res.message = "successfully disabled individual mode";
   res.success = static_cast<unsigned char>(true);
   return true;
 }
 
+bool flipperModeFLEnable(std_srvs::Trigger::Request& req, std_srvs::Trigger::Response& res)
+{
+  if (individual_mode == true)
+  {
+    flipper_mode_fl = true;
+    res.message = "successfully enabled flipper mode front left";
+    res.success = static_cast<unsigned char>(true);
+    return true;
+  }
+  return false;
```

Comment:
Maybe to be more friendly with the user, in the response you could tell that the call failed because the individual_mode wasn't enable ?